Packs/CheckpointFirewall/Playbooks/Checkpoint_-_Publish&Install_configuration.yml

```diff
@@ -0,0 +1,780 @@
+      description: Filter if Checkpoint install policy successful.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "20"
+      "yes":
+      - "3"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: CheckPoint
+                accessor: install-policy.task-id
+            iscontext: true
+          right:
+            value:
+              simple: succeeded
+    view: |-
+      {
+        "position": {
+          "x": -970,
+          "y": 2470
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "23":
+    id: "23"
+    taskid: 2771e1d3-d1c8-464a-8333-58dbc7819fba
+    type: condition
+    task:
+      id: 2771e1d3-d1c8-464a-8333-58dbc7819fba
+      version: -1
+      name: Filter TaskID
+      description: Filter TaskID to validate only new push configuration TaskID.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "12"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: CheckPoint
+                accessor: install-policy.task-id
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": -680,
+          "y": 1910
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "24":
+    id: "24"
+    taskid: 50c937c9-d4d0-4a92-8807-bbab08504154
+    type: condition
+    task:
+      id: 50c937c9-d4d0-4a92-8807-bbab08504154
+      version: -1
+      name: Filter TaskID
+      description: Filter TaskID to validate only new push configuration TaskID.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "9"
+      "yes":
+      - "13"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: CheckPoint
+                accessor: publish.task-id
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": -200,
+          "y": 10
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "5_7_yes": 0.51
+    },
+    "paper": {
+      "dimensions": {
+        "height": 3935,
+        "width": 1610,
+        "x": -970,
+        "y": -560
+      }
+    }
+  }
+inputs:
+- key: sid
+  value: {}
+  required: true
+  description: SID - Session unique identifier as it returned by the login request
+    required for publish changes. done by this SID will be seen by all users only
+    after publish is called and finish.
+outputs: []
```

Comment:
Consider adding outputs in case the playbook will be used as a subplaybook, for example an output to specify if the there was a policy update and if it was successful or failed.